File: ios/Harakat Lens/Services/Quran/SurahName.swift
//
//  SurahName.swift
//  Harakat Lens

import Foundation

struct SurahName: Decodable, Identifiable, Hashable {
    let number: Int
    let english: String
    let transliteration: String
    let arabic: String

    var id: Int {
        number
    }
}

File: android/app/src/main/java/dev/veeso/harakatlens/arabic/quran/SurahName.kt
package dev.veeso.harakatlens.arabic.quran

/**
 * Ported verbatim from the Harakat-Lens iOS app and the already-complete iOS
 * `ArabicExample/Quran/SurahName`.
 */
data class SurahName(
    val number: Int,
    val english: String,
    val transliteration: String,
    val arabic: String,
) {
    val id: Int get() = number
}
